WaterLog_Backend/Procedures.cs

```diff
@@ -1204,8 +1204,7 @@ public List<MonitorHeat> getMonitorsFaultLevels()
         public async Task<DataPoints<DateTime, double>> getTankGraph(int tankId)
         {
             var dailyTank = await _db
-                            .TankReadings
-                            .Where(a => a.TimeStamp.Day == DateTime.Now.Day && a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year && a.TankMonitorsId==tankId)
+                            .TankReadings.Where(a => a.TimeStamp.Day == DateTime.Now.Day && a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year && a.TankMonitorsId==tankId)
```

Comment:
update this line to be something like this 
```
var dailyTank = await _db
	.TankReadings.Where(a => 
		((a.TimeStamp.Day == DateTime.Now.Day) && (a.TimeStamp.Month == DateTime.Now.Month)) &&
		((a.TimeStamp.Year == DateTime.Now.Year) && (a.TankMonitorsId==tankId)))
	.GroupBy(b => b.TimeStamp.Day)
	.ToListAsync();
```

Reply:
Please take careful note of the use of `( )` and how I grouped them logically. 
Also make sure it still works as expected

Reply:
done